config/config.default.yaml

```diff
@@ -84,7 +84,7 @@ co2_budget:
 
 # docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#electricity
 electricity:
-  voltages: [200., 220., 300., 380., 500., 750.]
+  voltages: [220., 300., 330., 380., 400., 500., 750.]
   base_network: osm-prebuilt
   osm-prebuilt-version: 0.4
```

Comment:
why is the osm-prebuilt-version still at 0.4?

Reply:
updated now :)